cmd/hypper/upgrade.go

```diff
@@ -0,0 +1,193 @@
+/*
+Copyright The Helm Authors.
```

Comment:
It looks like there are changes from the helm code here, can you please add SUSE to the copyright.
e.g. 
```
Copyright The Helm Authors, SUSE.
```

That way it is easier for us to track changed code in maintenance.

Reply:
I dont think there are any changes to helm code here...yet.

On my next commit there should be changes and I will add that.

Reply:
:+1:

Reply:
That should be "SUSE LLC" :wink: